src/app/lib/utilities/passport/index.js

```diff
@@ -28,3 +28,19 @@ export const isValidPassportHome = (
 
   return passportHomesOverrideArray.some(matchesPassportHome);
 };
+
+export const getCanonicalUrl = pageData => {
+  const canonicalUrl = pathOr(
+    null,
+    ['metadata', 'locators', 'canonicalUrl'],
+    pageData,
+  );
+  return canonicalUrl
+    ? canonicalUrl.replace(/.*(bbc.com|localhost:\d*)/, '')
```

Comment:
or is this always returned from BFF with .com?

Reply:
Good question! Yes the BFF returned the canonical URL with .com for the 3 examples we looked at. Though I'm not sure how to check if this will always be the case.

Reply:
I would maybe add in bbc.co.uk here as well. I think its rare, but we have seen it in the past in other fields where we'd expect it to be bbc.com.